styles/stylesheet.css

```diff
@@ -0,0 +1,353 @@
+
+/*Style for header starts here*/
+header {
+  background-image: url("../images/header-image.jpg");
+  background-color: #0f3357;
+  color: white;
+  font-weight: bold;
+  text-align: left;
+}
+
+.buy {
+  width: 5%;
+  margin-top: 1%;
+  margin-right: 1%;
+  margin-left: 1%;
+  padding-top: 2%;
+  padding-left: 85%;
+}
+
+.cart {
+  width: 40%;
+  height: 20%;
+}
+
+.checkout {
+  width: 100%;
+  height: 50%;
+  background-color: #39c639;
+  text-align: center;
+  padding: 5% 20% 5% 20%;
+  color: white;
+}
+
+a {
+  text-decoration: none;
+}
+
+.logo {
+  margin-left: 10%;
+  margin-bottom: 5%;
+  color: #39c639;
+  width: 30%;
+  font-size: 110%;
+  font-family: 'Open Sans', sans-serif;
+  font-style: italic;
+}
+
+.header_nav {
+  background-color: #f2f2f2;
+  color: #0f3357;
+  padding-left: 15%;
+  padding-top: 1%;
+  padding-bottom: 1%;
+  font-size: 120%;
+}
+
+/*End header style and begin index page style*/
+
+.intro {
+  float: left;
+  width: 40%;
+  margin-left: 15%;
+  margin-top:2%;
+  font-size: 100%;
+}
+
+@media screen and (max-width: 768px) {
+  .intro {
+    width: 80%;
+    margin-left 15%;
+  }
+}
+
+p {
+  font-family: 'Lato', sans-serif;
+}
+
+.golden {
+  display: block;
+  float: none;
+  width: 100%;
+  padding-right: 5%;
+  float: left;
+  margin-bottom: 5%;
+}
+
+h2 {
+  color: #0f3357;
+  font-size: 200%;
+  font-family: 'PT Serif', serif;
+}
+
+a {
+  color: inherit;
+}
+
+aside {
+  display: block;
+  float: right;
+  width: 20%;
+  margin-right: 15%;
+  font-size: 80%;
+}
+
+.aside_image {
+  width: 80%;
+}
+
+@media screen and (max-width: 991px) {
+  .aside_image {
+    width: 65%;
+    float: center;
+  }
+}
+
+@media screen and (max-width: 991px) {
+  aside {
+    width: 15%;
+    float: right;
+    margin-left: 5%;
+  }
+}
+
+@media screen and (max-width: 767px) {
+  aside {
+    display: none;
+  }
+}
+
+.flex-container-home {
+  display: flex;
+  float: left;
+  width: 40%;
+  margin-left: 15%;
+  margin-top:2%;
+  flex-flow: row wrap;
+  justify-content: space-between;
+  align-content: space-between;
+  font-size: smaller;
+}
+
+@media screen and (max-width: 991px) {
+  .flex-container-home {
+    width: 50%;
+    float: left;
+  }
+}
+
+@media screen and (max-width: 767px) {
+  .flex-container-home {
+    float: center;
+    width: 50%;
+  }
+}
+
+
+.box-home {
+  width: 30%;
+  margin-top: 1%;
+  margin-bottom: 1%;
+  border: 2px solid #e6e6e6;
+  padding: 1%;
+}
+
+@media screen and (max-width: 991px) {
+  .box-home {
+    width: 39%;
+  }
+}
+
+@media screen and (max-width: 576px) {
+  .box-home {
+    width: 100%;
+    float: center;
+  }
+}
+
+.dog_photo {
```

Comment:
I think you are trying to make the dog images larger here. You need the container to be larger to give them more space. Edit the width of .flex-container-home instead

Reply:
Thank you, that seemed to take care of my wasted white space issue.